spec/unit/resource/rhsm_register_spec.rb

```diff
@@ -158,6 +158,48 @@
         end
       end
 
+      context "when a server_url is provided" do
+        it "returns a command containing the server url" do
+          allow(resource).to receive(:server_url).and_return("https://fqdn.example")
```

Comment:
As a style thing I'd prefer `resource.server_url "https://fqdn.example"` to really set the state on the resource over stubbing the response to the resource (but it seems the whole rest of this file was written that way so the whole thing should really get fixed and you're just being consistent).